yarn-project/end-to-end/src/cli_docs_sandbox.test.ts

```diff
@@ -96,6 +96,7 @@ Rollup Address: 0x0dcd1bf9a1b36ce34237eeafef220932846bcd82
     const docs = `
 // docs:start:example-contracts
 % aztec-cli example-contracts
+AvmTestContractArtifact
 BenchmarkingContractArtifact
 CardGameContractArtifact
 ChildContractArtifact
```

Comment:
Sandbox test fails without this. Should we filter the `aztec-cli example-contract` command so it doesn't show the avm test contract?

Reply:
this is probably the move

Reply:
done